.github/CODEOWNERS

```diff
@@ -64,3 +64,8 @@
 /airbyte-integrations/connectors/source-salesforce/ @airbytehq/dev-extensibility
 /airbyte-integrations/connectors/source-shopify/ @airbytehq/dev-extensibility
 /airbyte-integrations/connectors/source-stripe/ @airbytehq/dev-extensibility
+
+# Breaking changes detection
+# This rule ensures that breaking changes to connectors (identified by migrations files) 
+# are reviewed by the breaking-changes-reviewers team to maintain compatibility
+docs/integrations/**/*-migrations.md @airbytehq/breaking-changes-reviewers
```

Comment:
Do we have CI rules that if there is a breaking change, we need to update a migration file? If so, I really love this solution as it seems way easier than the previous one!

Reply:
@maxi297 - Yes, CI pre-release checks will fail if any breaking change does not have a matching entry in the respective `-migrations.md` file. Hence, new breaking changes require modifying the repective file (triggering this requirement).

Reply:
Awesome, I'm in love ❤️ . Approving right now